boards/nordic/nrf54h20dk/doc/index.rst

```diff
@@ -0,0 +1,154 @@
+.. _nrf54h20dk_nrf54h20:
+
+nRF54H20 DK
+###########
+
+Overview
+********
+
+.. note::
+
+   All software for the nRF54H20 SoC is experimental and hardware availability
+   is restricted to the participants in the limited sampling program.
+
+The nRF54H20 DK is a single-board development kit for evaluation and development
+on the Nordic nRF54H20 System-on-Chip (SoC).
+
+The nRF54H20 is a multicore SoC with:
+
+* an Arm Cortex-M33 core with DSP instructions, FPU, and Armv8-M Security
+  Extensions, running at up to 320 MHz, referred to as the **application core**
+* an Arm Cortex-M33 core with DSP instructions, FPU, and Armv8-M Security
+  Extensions, running at up to 256 MHz, referred to as the **radio core**.
+
```

Comment:
Shouldn't this describe ppr?

Reply:
we should

Reply:
Yes, info about PPR and its targets will be added.

Reply:
@nordicjm How about adding it in a follow-up PR together with the picture that is not available yet?

Reply:
> @nordicjm How about adding it in a follow-up PR together with the picture that is not available yet?

Works for me